docs/service-accounts.md

```diff
@@ -0,0 +1,76 @@
+# Kubernetes Service Accounts
+
+Knative Eventing has many pieces. Many of those pieces run Kubernetes `Pods`,
+meaning they run as Kubernetes `ServiceAccounts`. This document describes
```

Comment:
nitpick: should we use `ServiceAccount`s instead of `ServiceAccounts`?

Reply:
I've used both. I don't really have a preference between the two, let's pick one style and use it consistently.

Reply:
Switched to the K8s style of not using the back tics.